CHANGELOG.md

```diff
@@ -1,14 +1,22 @@
 ## Changelog
 
-#### V0.1.0
-- Initial Release
-
-#### V0.1.1
-- Add some documentation fixes
+#### V0.1.5
+- Add `find_or_create` for Actions
```

Comment:
Reversed the order of your changelog, not sure on what you personal preference is. I got the changes from your commit history.